docs/get-started/os/rt-thread.md

```diff
@@ -17,29 +17,25 @@
 - Supports all mainstream compiling tools such as GCC, Keil and IAR.
 - Supports a wide range of <a href="https://www.rt-thread.io/board.html">architectures and chips</a>.
 
-
 ## How to run LVGL on RT-Thread?
 
 [中文文档](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/packages-manual/lvgl-docs/introduction)
 
 LVGL has registered as a [software package](https://packages.rt-thread.org/en/detail.html?package=LVGL) of RT-Thread. By using [Env tool](https://www.rt-thread.io/download.html?download=Env) or [RT-Thread Studio IDE](https://www.rt-thread.io/download.html?download=Studio), RT-Thread users can easily download LVGL source code and combine with RT-Thread project. RT-Thread community has port LVGL to several BSPs:
 
-| BSP                                                          | Note |
-| ------------------------------------------------------------ | ---- |
-| [QEMU simulator](https://github.com/RT-Thread/rt-thread/tree/master/bsp/qemu-vexpress-a9/applications/lvgl) |      |
-| [Visual Studio simulator](https://github.com/RT-Thread/rt-thread/tree/master/bsp/simulator/applications/lvgl) |      |
-| [Nuvoton numaker-iot-m487](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/numaker-iot-m487/applications/lvgl) |      |
-| [Nuvoton numaker-pfm-m487](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/numaker-pfm-m487/applications/lvgl) |      |
-| [Nuvoton nk-980iot](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/nk-980iot/applications/lvgl) |      |
-| [Nuvoton numaker-m2354](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/numaker-m2354/applications/lvgl) |      |
-| [Nuvoton nk-n9h30](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/nk-n9h30/applications/lvgl) |      |
-| [Nuvoton numaker-m032ki](https://github.com/RT-Thread/rt-thread/tree/master/bsp/nuvoton/numaker-m032ki/applications/lvgl) |      |
-| [NXP imxrt1060-evk](https://github.com/RT-Thread/rt-thread/tree/master/bsp/imxrt/imxrt1060-nxp-evk/applications/lvgl) |      |
-| [STM32L475 pandora](https://github.com/RT-Thread/rt-thread/tree/master/bsp/stm32/stm32l475-atk-pandora/applications/lvgl) |      |
-| [STM32F407 explorer](https://github.com/RT-Thread/rt-thread/tree/master/bsp/stm32/stm32f407-atk-explorer/applications/lvgl) |      |
-| [STM32F469 Discovery](https://github.com/RT-Thread/rt-thread/tree/master/bsp/stm32/stm32f469-st-disco/applications/lvgl) |      |
-| [Raspberry PICO](https://github.com/RT-Thread/rt-thread/tree/master/bsp/raspberry-pico/applications/lvgl) |      |
-
+| BSP                                                                                                                               | Note                                                                                                                                              |
```

Comment:
It seems there are BSPs in both columns.

```suggestion
| BSP                                                                                                                               | BSP                                                                                                                                              |
```

Reply:
Yes, I intentionally did that. Because there is no note for BSPs, and there are more BSPs supported.

Reply:
I don't get it. It seems there are the same kind of BSPs in both columns: links to 
`https://github.com/RT-Thread/rt-thread/tree/master/bsp/<TARGET>/applications/lvgl`, but the first column is called BSP, and the second is `Note`

Reply:
I leave a blank for Note, but there is note for each bsp in fact. So the note can be removed. Two column are all BSPs to reduce the rows of the document. Just like this: https://github.com/RTduino/RTduino#11-rt-thread-bsps-which-have-supported-rtduino

Reply:
> I leave a blank for Note

Sounds good!

Reply:
emm no, I mean, I leaved a blank for note for every BSP, but I found that those blank notes are meaningless, so I remove it now.

Reply:
Do you want to leave that notes? or we can remove those

Reply:
I still don't get it, but I'm sure you know now better why it should be this way.  :slightly_smiling_face: 

So I merge it.